code/src/components/Parent.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-unused-vars */
```

Comment:
Is this disabled because useEffect was never used in this component?

Reply:
No i think it's `const postNewThought` that i never used, but it's weird because i thought this function was needed to "clean up" the form after submit, but somehow it still works 🤔